.github/workflows/check-docs.yml

```diff
@@ -0,0 +1,68 @@
+name: check docs
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+jobs:
+  check:
+    name: check docs
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: '0'
+    - run: git fetch --depth=1 origin +refs/tags/*:refs/tags/* +refs/heads/*:refs/remotes/origin/*
+    - name: Set up Python 3.7
```

Comment:
If possible we should change here to Python 3.8 to be consistent with the other workflows?

Reply:
I wanted to test on the oldest supported version in case we were unknowingly relying on later features; I don't think consistency with other workflows is really important here.

Reply:
Okay. Thank you for your answer.